src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -209,6 +209,10 @@ protected virtual void ExtractGeneralParameters(List<string> keys, IUnformattedD
       public IEnumerable<ParsedDataSet> Parse(IUnformattedData data,
          IReadOnlyList<ColumnInfo> columnInfos)
       {
+         var missingColumns = Parameters.Where(p => p.ComesFromColumn() && data.GetColumnDescription(p.ColumnName) == null).Select(p => p.ColumnName);
+         if (missingColumns.Any())
```

Comment:
You are iterating here already . So call the toList before on line 212 and not on 214.

Reply:
done